pkgs/by-name/cd/cd2netmd-gui/package.nix

```
@@ -0,0 +1,114 @@
+{
+  atracdenc,
+  bash,
+  cmake,
+  copyDesktopItems,
+  fetchFromGitHub,
+  ffmpeg,
+  lib,
+  libcdio,
+  libcdio-paranoia,
+  libgcrypt,
+  libgpg-error,
+  libusb1,
+  makeDesktopItem,
+  netmdplusplus,
+  nlohmann_json,
+  pkg-config,
+  qt5,
+  stdenv,
+  taglib,
+}:
+stdenv.mkDerivation rec {
+  pname = "cd2netmd-gui";
+  version = "2.1.6";
+
+  src = fetchFromGitHub {
+    owner = "Jo2003";
+    repo = "cd2netmd_gui";
+    tag = "v${version}";
```

finalAttrs should be used here to make overriding easier
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "cd2netmd-gui";
  version = "2.1.6";

  src = fetchFromGitHub {
    owner = "Jo2003";
    repo = "cd2netmd_gui";
    tag = "v${finalAttrs.version}";
```

---

Good idea. Implemented `finalAttrs` in place of `rec` and referenced `finalAttrs` also in some other parts of the derivation, like some variables in `postPatch`